libs/qCC_glWindow/src/ccGLWindow.cpp

```diff
@@ -1169,7 +1164,11 @@ bool ccGLWindow::event(QEvent* evt)
 		evt->accept();
 	}
 	return true;
-
+#else
+	case QEvent::Resize:
+	{
+		update();
+	}
```

Comment:
you'll still need to add a 'break' statement.

Reply:
My bad, will add it right away.